modules/values.tmpl.yaml

```diff
@@ -465,7 +471,16 @@ minio:
 secrets-store-csi-driver: {}
 
 %{ if traefik.enable }
-traefik: {}
+traefik:
+  middlewares:
+    redirections:
+      # a lowercase RFC 1123 subdomain must consist of lower
+      # case alphanumeric characters, '-' or '.', and must start
+      # and end with an alphanumeric character
+      withclustername:
+        permanent: true
```

Comment:
does this do a rewrite or does it return a 301?

Reply:
@raphink This is a 301 redirect.

Reply:
@modridi maybe a temporary redirect with a 302 ? If we switch from blue to green, browser will keep the previous redirection and it will not work.

Reply:
@Vampouille done. Thanks!